models/cartDao.js

```diff
@@ -70,9 +70,74 @@ const getProductById = async (productId) => {
   return product;
 };
 
+const postProductsInCart = async (userId, productId, quantity) => {
+    try {
+        const carts = await dataSource.query(
+          `INSERT INTO 
+            carts (
+            user_id, 
+            product_id, 
+            quantity
+            ) VALUES (?,?,?)`,
+          [userId, productId, quantity]
+        );
+    
+        return carts;
+      } catch (error) {
+        const err = new Error("INVALID_INPUT_DATA");
+        err.statusCode = 400;
+        throw err;
+      }
+    };
+const patchProductsInCart = async(user_id,product_name, quantity)=>{
+    try{
+        if(quantity == 0){
+            console.error("0개 미만으로는 숫자를 변경할 수 없습니다.",error)
+            error.statusCode = 400;
+            throw error;
+        }
```

Comment:
아래의 내용처럼 수정을 고려해주세요! 이전 리뷰 내용이 미반영상태처럼 보여집니다~

```suggestion
const patchProductsInCart = async(user_id,product_name, quantity)=>{
    try{
        if(quantity == 0){
            const error = new Error("INVALID QUANTITY REQUEST")
            error.statusCode = 400;
            throw error;
        }
```